### PR TITLE
Remove the need to edit code for compilation when not on Windows

### PR DESCRIPTION
**Pull request by Zacho2**

### CODE REVIEW

File: src/Picocrypt.go [hunk 1]
```diff
@@ -957,7 +949,9 @@ func onDrop(names []string) {
 		return
 	}
 	if tab == 2 {
-		go shred(names, true)
```
Comment (Zacho2) [inline src/Picocrypt.go:960]: This is racy since names[0]` may modified down below so I took a copy.

**Comment by HACKERALERT** [comment]
This is excellent! Thank you!

**Comment by HACKERALERT** [comment]
Edit: there were some issues with the immediate mode GUI after the merge, so I am afraid I will have to revert. Thanks for the effort though!

**Comment by HACKERALERT** [comment]
I'm pretty busy so I don't have the time to debug the issues and fix them. If you get things working, you're free to make another PR so that I can merge the working version.

**Pull request merged**